### PR TITLE
[ADD] hr_holidays_compute_days

### PR DESCRIPTION
**Pull request by saltonmassally**
compute leave days taking into consideration rest days and public holidays

### CODE REVIEW

File: hr_holidays_compute_days/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/hr
+* add the path to this repository in your configuration (addons-path)
+* update the module list
+* search for "Employee Compute Leave Days" in your addons
+* install the module
+
```
Comment (max3903) [inline hr_holidays_compute_days/README.rst:16]: No need to mention these steps. Once merged, your module will be available on Odoo Apps.

File: hr_holidays_compute_days/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Configuration
+=============
+
+Go to your holiday status and specify what leaves should not include rest days
+and holidays in computation
+
+Usage
+=====
+
+
```
Comment (max3903) [inline hr_holidays_compute_days/README.rst:26]: A scenario using the computation would be nice here.

Please refer to the template to include the button to try your module on Runbot:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: hr_holidays_compute_days/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+###############################################################################
+
+{
+    'name': 'Employee Compute Leave Days',
+    'version': '8.0.1.0.0',
+    'category': 'Human Resources',
+    'license': 'AGPL-3',
+    'summary': 'Computes the actual days for which employee will be on leave '
+               'taking into account both rest days and public holidays',
+    'author': 'Salton Massally<smassally@idtlabs.sl>',
```
Comment (max3903) [inline hr_holidays_compute_days/__openerp__.py]: Summary might be a little too long.

Please mention the OCA as an additional author.

**Comment by saltonmassally** [comment]
@max3903 thanks for reviewing... will apply your suggestions

**Comment by dreispt** [comment]
Please fix Maxime's comments. Other than that :+1:

**Comment by feketemihai** [comment]
Could you change the readme {} variables in links to the correct ones.

Coding is ok, but i want to test it on runbot after the hr_public_holidays module will be merged, so i think we should wait untill runbot is green and test ok.

**Comment by dreispt** [comment]
@tarzan0820 ping!

**Comment by saltonmassally** [comment]
@feketemihai thanks for reviewing, will clean up the hr_public_holidays this weekend

**Pull request closed without merging**